package.json

```diff
@@ -73,7 +73,7 @@
     ]
   },
   "devDependencies": {
-    "@genesislcap/rollup-builder": "14.30.1", 
+    "@genesislcap/rollup-builder": "14.30.1",
```

Comment:
question:

do we have `14.62.1` for this package?

Reply:
Yes, updated